packages/table/src/interactions/_interactions.scss

```diff
@@ -86,18 +86,18 @@ $resize-handle-dragging-color: $pt-intent-primary !default;
 
   &.bp-table-resize-vertical {
     top: 0;
-    right: -$resize-handle-padding;
-    bottom: -$cell-border-width;
+    right: 0;
+    bottom: -1px;
     cursor: $resize-vertical-cursor;
-    width: $resize-handle-padding * 2;
+    width: 5px;
```

Comment:
Can we name this value since it appears twice? Would `$resize-handle-trigger-width` or `$resize-handle-target-width` be appropriate names?